layouts/default/alice/default_alice/keymap.c

```diff
@@ -0,0 +1,24 @@
+#include QMK_KEYBOARD_H
```

Comment:
There should be a license header here.

Sample:

https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15